api/requirements.in

```diff
@@ -17,3 +17,4 @@ psycopg2
 python-dateutil
 requests
 whitenoise
+pdfminer.six==20170720
```

Comment:
The requirements.in file generally shouldn't contain explicit versions (that's the role of the locked requirements.txt file). Can we follow that rule here or is there something special about pdfminer?

Reply:
Oops, I guess I'm just used to maintaining `requirements.txt` files directly rather than `requirements.in` files... and I noticed that Django pinned itself to a particular major version here, so it felt safer to keep pdfminer pinned as well.  No real reason aside from old habits though, so I will fix this in my next PR!